test/server.py

```diff
@@ -78,7 +80,7 @@ def _start_server(self):
         sock.close()
 
     def _wrap_socket(self, sock):
-        raise NotImplementedError()
+        return self.cxt.wrap_socket(sock, server_side=True)
```

Comment:
Is it OK? What was the purpose of this method? I couldn't find any references to it